LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 NodeSecure
+Copyright (c) 2023 NodeSecure
```

Comment:
```suggestion
Copyright (c) 2021-2023 NodeSecure
```
Shouldn't it be a range date includes starting date instead of the current year ? (I'm not sure, I guess both are ok but looking at some project, it usually includes the range date ^^)